pkg/scheduler/apis/config/testing/defaults/defaults.go

```diff
@@ -151,8 +151,41 @@ var PluginConfigsV1beta2 = []config.PluginConfig{
 	},
 }
 
-// PluginsV1beta3 default set of v1beta3 plugins.
+// PluginsV1beta3 is the set of default v1beta3 plugins (before MultiPoint expansion)
```

Comment:
this will always be updated though, so I would replace "before" with "unrolled"

Reply:
this one not addressed yet

Reply:
@ahg-g could you clarify the wording you'd like here? Not sure I understand... `// PluginsV1beta3 is the set of default v1beta3 plugins (unrolled MultiPoint expansion)`?

Reply:
ok, perhaps I misunderstood this one, I guess the point is that those are after the multipoint is unrolled, if yes, then I don't think the original comment is correct, right?

Reply:
So `PluginsV1beta3` is the actual default, *before* the MultiPoint is unrolled (in other words, when all plugins are still just enabled in MultiPoint)

`ExpandedPluginsV1beta3` (below) is the default set of plugins *after* being unrolled.

So I think the original comment here is correct, if that makes sense?

Reply:
ok, sg